Automattic/simplenote-ios
# NavigationBar Blur Adjustments

### Fix
In this PR we're adjusting the NavigationBar Blur Effect so that it's inactive until the underlying ScrollView is scrolled.

We're only applying this over to:

- Notes List
- Note Editor

@aerych Me again!!! promise this one is fun!!!
Thanks in advance!!!

### Test
1. Log into your account
2. Verify the Notes List's navigation bar's background color matches with the notes list BG
3. Scroll upwards
4. Verify the Blur gradually fades in, as you scroll
5. Press over any note
6. Verify the navigation bar's background color matches the editor
7. Scroll upwards
8. Verify the blur slooowly fades in!

### Release
These changes do not require release notes.


---

### Demo!

![RPReplay_Final1573174324](https://user-images.githubusercontent.com/1195260/68440448-dc93dc80-01a9-11ea-986f-96a11d9e62e8.gif)
![RPReplay_Final1573174493](https://user-images.githubusercontent.com/1195260/68440449-dd2c7300-01a9-11ea-9f4a-5b3444ea3331.gif)
